NorthstarDedicatedTest/dllmain.cpp

```diff
@@ -158,13 +158,34 @@ bool LoadPlugins()
 		{
 			spdlog::error("{} does not have a version number in its manifest", pathstring);
 			continue;
-			// spdlog::info(manifestJSON["version"].GetString());
+		}
+		if (!manifestJSON.HasMember("run_on_client") || !manifestJSON.HasMember("run_on_server"))
```

Comment:
Shouldn't it be `&&` instead of `||`

Cause currently a plugin that only specifies client (like DiscordRPC) will hit true on the second condition and enter the `if` clause

Reply:
No, this is checking if either of them *doesnt* exist

Reply:
Ah so it makes sure that keys are set regardless of content (`true`/`false`)? In that case I misread and it should be fine as is ^^